doc/commands/module.8.rst

```diff
@@ -46,31 +48,44 @@ Subcommands
 ``info``
     | Print details about module streams.
 
-``provides``
-    | Print module and module profile the specified packages come from.
-
 ``enable``
     | Enable module streams and make their packages available.
 
 ``disable``
     | Disable modules including all their streams.
 
-``switch-to``
-    | Enable different module streams, upgrade their profiles and distro-sync packages.
-
 ``reset``
     | Reset module state so it's no longer enabled or disabled.
 
-``install``
-    | Install module profiles, including their packages.
-
-``remove``
-    | Remove installed module profiles including their packages.
-
 
 Options
 =======
 
+``--skip-broken``
+    | Used with ``enable`` command to resolve any dependency problems by removing items that are causing problems from the transaction.
+
+``--skip-unavailable``
+    | Used with ``enable``, ``disable`` and ``reset`` commands to allow skipping modular dependencies that are not possible to resolve.
+    | All remaining items will be processed.
```

Comment:
This sounds like this would allow skipping available modules with unavailable dependencies. But it should be the same as for RPM package: --skip-unavailable should skip modules that are not available in the repositories and --skip-broken should skip modules with dependency problems.